src/xdocs-examples/java/com/puppycrawl/tools/checkstyle/checks/naming/MethodTypeParameterNameCheckExamplesTest.java

```diff
@@ -34,10 +36,11 @@ protected String getPackageLocation() {
     @Test
     public void testExample1() throws Exception {
         final String[] expected = {
-
```

Comment:
Remove
`@Disabled("until https://github.com/checkstyle/checkstyle/issues/13345`

Reply:
Before you push to GitHub, I make sure that `mvn clean verify` pass on your local.

After push recheck CI is failures

Reply:
I ran 'mvn clean verify' locally before committing but it failed consistently because it could not find resource 'https://raw.githubusercontent.com/oasis-tcs/sarif-spec/master/Documents/CommitteeSpecifications/2.1.0/sarif-schema-2.1.0.json'. 

I just thought to try it in CI, so I committed.

Reply:
@romani The Requested change has been made, Please approve for merge.